.github/workflows/template_build.yml

```diff
@@ -27,7 +30,7 @@ jobs:
     name: Build
     runs-on: ubuntu-22.04
     container:
-      image: nycplanning/dev:latest
+      image: nycplanning/dev:${{ inputs.image_tag || 'latest' }}
```

Comment:
This was needed in the first commit since it's called by the `test` job. But ended up going with the same approach in commit 2